lib/international_optin_people.rb

```diff
@@ -0,0 +1,19 @@
+INTERNATIONAL_OPTIN_FACILITATORS = [
```

Comment:
This should be contained in a named module, which can then be included where needed. 

Then, since `dashboard/lib` is [autoloaded](https://github.com/code-dot-org/code-dot-org/blob/bd5abfe88ff4e4f3eb8d2949c100b34c8d95f12b/dashboard/config/application.rb#L116) we won't need to explicitly [require](https://github.com/code-dot-org/code-dot-org/pull/23235/files#diff-c89624673ce6bf54fdb9c2f8cbb32055R16) this file, and these constants and methods will be contained in the module and applied to the imported class. Better containment, better style :)